changelog.d/4089.feature

```diff
@@ -0,0 +1,4 @@
+Currently, the Docker image only listens on ipv4, making it unusable on ipv6-enabled setups.
```

Comment:
ok, sorry: this file is what ends up in the changelog, so needs to be a succinct one-line summary. see https://github.com/matrix-org/synapse/blob/master/CHANGES.md#synapse-0337rc1-2018-10-15 for some examples, but basically, you want something more like "Configure the Docker image to listen on IPv6 addresses".

I'm sorry the CONTRIBUTING.rst doesn't make that clearer :/

Reply:
It's done :)